src/main/resources/logback-spring.xml

```diff
@@ -90,8 +93,11 @@
                 <maskPattern>"access_token":"([^"]+)"</maskPattern>
                 <maskPattern>Authorization: Bearer (.+)</maskPattern>
 		<!-- Mask GitHub tokens -->
-		<maskPattern>https://([_a-zA-Z0-9]+)@github.com</maskPattern>
-		<maskPattern>Authorization: token ([_a-zA-Z0-9]+)$</maskPattern>
+                <maskPattern>https://([_a-zA-Z0-9]+)@github.com</maskPattern>
+                <maskPattern>https://([_a-zA-Z0-9]+)@dev.azure.com</maskPattern>
+                <maskPattern>https://([_a-zA-Z0-9:.\\-]+)@bitbucket.org</maskPattern>
+                <maskPattern>https://([_a-zA-Z0-9:.\\-]+)@gitlab.com</maskPattern>
```

Comment:
I'd suggest you generalize this, currently it doesn't support self-hosted instances, plus I don't think there is a single web site where you'd want to show the authentication token anyway


```
<maskPattern>https://([_a-zA-Z0-9:.\\-]+|)@.+</maskPattern>
```